src/components/products/Products.css

```diff
@@ -0,0 +1,134 @@
+.display-flex-filter {
+  background-color: var(--background-color);
+  display: flex;
+  flex-direction: row;
+  width: 30%;
+  margin: 3rem 2rem;
+  padding: 3rem;
+}
+.grid-left-filter {
+  width: 100%;
+}
+.filter-title {
+  display: flex;
+  justify-content: flex-start;
+  padding: 2rem 1rem 5px 1rem;
+}
+.left-filter-title {
+  font-size: 1.8rem;
+  font-weight: 500;
+}
+.right-clear-btn {
+  color: var(--primary-color);
+  font-size: 1.7rem;
+  background-color: var(--background-color);
+  height: 2rem;
+  width: 12rem;
+  margin: 5px;
+  padding: 0;
+}
+.right-clear-btn:hover {
+  text-decoration: underline;
+}
+.filter-section {
+  overflow-y: auto;
+  padding: 5px 1rem;
+}
+.filter-divider {
+  background-color: var(--navigation-color);
+  height: 1px;
+  width: 12rem;
+  margin-top: 2px;
+  margin-bottom: 2px;
+}
+.filter-section-title {
+  padding: 2rem 1rem 5px 0rem;
+}
+.filter-item {
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+}
+.filter-item:hover {
+  color: var(--navigation-color);
+  font-weight: 400;
+}
+.filter-radio,
+.filter-checkbox {
+  margin-right: 10px;
+}
+
+/* products-grid */
+.grid-3-column-layout {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
```

Comment:
here you could use something like `grid-template-column:repeat(5,280px)` to make it more responsive with other required css